docs/source/reference/api.rst

```diff
@@ -52,6 +52,62 @@ sky.autostop
 
 .. autofunction:: sky.autostop
 
+
+Job Queue API
+-----------------
+
+sky.queue
+~~~~~~~~~~
+
+.. autofunction:: sky.queue
+
+sky.job_status
+~~~~~~~~~~~~~~~~
+
+.. autofunction:: sky.job_status
+
+
+sky.tail_logs
+~~~~~~~~~~~~~~~~~
+
+.. autofunction:: sky.tail_logs
+
+
+sky.download_logs
+~~~~~~~~~~~~~~~~~~~~~
+
+.. autofunction:: sky.download_logs
```

Comment:
Our manually annotated `(type)` can be removed

Reply:
Removed. Thanks!